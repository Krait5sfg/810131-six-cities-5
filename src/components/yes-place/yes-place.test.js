import React from 'react';
import renderer from 'react-test-renderer';
import YesPlace from './yes-place';

it(`YesPlace render correctly`, () => {
  const tree = renderer.create(<YesPlace
    offerCount={10}
    city={`Amsterdam`}
  />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

