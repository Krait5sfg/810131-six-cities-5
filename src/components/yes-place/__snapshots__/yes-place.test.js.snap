// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YesPlace render correctly 1`] = `
<section
  className="cities__places places"
>
  <h2
    className="visually-hidden"
  >
    Places
  </h2>
  <b
    className="places__found"
  >
    10
     places to stay in 
    Amsterdam
  </b>
</section>
`;
